Prime_Number/Prime_Number.c

```diff
@@ -0,0 +1,32 @@
+#include <stdio.h>
+#include <stdbool.h>
+ 
+bool Prime (int num, int i) 
+{ 
+    // Base cases 
+    if (i == 1) 
```

Comment:
Better to add base case as i <= 2, since 1 is not prime.

Reply:
Oops! I meant num <= 2, after which you can check for if num == 2, then return true else return false. But then you would require a separate base case for i.

Best would be to revert back to the case which you were having initially and add another base case `if (num <= 1) return false;` so that your code doesn't break.

Sorry for missing this earlier.

Reply:
Rest of the things are fine!